python/iceberg/hive/hive_table_operations.py

```diff
@@ -85,20 +89,20 @@ def commit(self, base, metadata):
 
             tbl.sd = storage_descriptor(metadata)
             metadata_location = tbl.parameters.get(BaseMetastoreTableOperations.METADATA_LOCATION_PROP, None)
-            base_metadata_location = base.metadataFileLocation() if base is not None else None
+            base_metadata_location = base.location if base else None
```

Comment:
I think this does need to be the metadata file location and not the table location.

Reply:
`metadataFileLocation` is the Java name, `location` is the Python name. Same value though.

Reply:
That seems confusing to me. How do you get the table's base location in Python?

Reply:
ahh, i see now. It appears that the Java and Python impls are a bit out of sync. Python doesn't use `file` at all, which is where java takes its location from. Will look at figuring out what happened there and update the PR this eve or tomorrow

Reply:
should be good to go now @rdblue, we had missed a property in `TableMetadata` in python